Adds Galaxy Zoo 3D Segmentation Data (#122)
MultimodalUniverse/MultimodalUniverse: DESIGN.md

}
```

### Segmentation Maps

The organization of the data is intended to allow segmentation maps to exist for any data structure. Meta data is included, as well as the source data for which the segmentaitons were developed (i.e. false color images of galaxies). The dimensions of the source data define the dimensions of the segmentation maps (segmentation arrays). An exmample for a galaxy segmentation map (from the `gz3d` [dataset](./scripts/gz3d/README.md)) looks like:

  ```python
    {
        'total_classifications': 60,
        'healpix': 1823,
        'ra': 357.58013916,
        'dec': -11.06652546,
        'image': array(512, 512, 3),
        'scale': 2.75000002e-05,
        'segmentation': {
            'class': ['center', 'star', 'spiral', 'bar'],
            'vote_fraction': [-1.        , -1.        ,  0.75      ,  0.51666667],
            'array': array(4, 512, 512)
            },
        'object_id': '1-100017'
    }
  ```

> Our main condition is that the data can be mapped back to the relevant survey, and so the segmentation array sizes must match the relevant data from which the maps were derived.

Author: ```suggestion
Our main condition for segmentation maps, is that the data can be _mapped back_ to the relevant survey. Specifically, the `segmentation.array` sizes and pixel scale must match the relevant data from which the maps were derived.
```

> Our main condition for segmentation maps, is that the data can be _mapped back_ to the relevant survey. Specifically, the `segmentation.array` sizes and pixel scale must match the relevant data from which the maps were derived.

Reviewer: ```suggestion
Our main condition for segmentation maps, is that the data can be _mapped back_ to the relevant survey. Specifically, the `segmentation.array` sizes and pixel scale must match the relevant data from which the maps were derived. Furthermore, the spatial coordinates of the pixels in the segmentation map should implicitly match the spatial coordinates in the relevant survey (which also implies that the segmentation maps and survey have the same orientation).
```

Author: ```suggestion
Our main condition for segmentation maps, is that the data can be _mapped back_ to the relevant survey. Specifically, the `segmentation.array` sizes and pixel scale must match the relevant data from which the maps were derived. For example, the spatial coordinates of the pixels in the segmentation map should directly match the spatial coordinates of the corresponding pixels in the relevant survey, (which also implies that respective arrays are oriented in the same way).
```
